### DISCUSSION TITLE
Guideline - csdiy.wiki

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#548] [Announcements]
# Guideline - csdiy.wiki

CS自学指南

https://csdiy.wiki/en/CS%E5%AD%A6%E4%B9%A0%E8%A7%84%E5%88%92/

**Comment by HenanLee** [comment]
good

**Comment by Mawfyy** [comment]
That's good, i mean this guideline are so complete

**Comment by AOisUsed** [comment]
Thanks for providing access links to these plenty of free online resources for cs students and a comprehensive guideline to self-learning. PKU is almost the top-tier university here but it seems that students there encounter the same problems as anywhere else in here. Thank you again.
